### ISSUE TITLE
Expression and code controls should be accessible

### ISSUE THREAD

**Issue by nmgokhale** [elyra-ai/canvas#497]
Fix a11y violations in these controls -

[Expression control](https://wdp-common-canvas-dev.stage1.mybluemix.net/#/properties#--expression) -
![Screen Shot 2021-02-22 at 12 38 19 PM](https://user-images.githubusercontent.com/25124000/108767150-e52d3500-750a-11eb-9739-c0c3d87eac60.png)

[Code control](https://wdp-common-canvas-dev.stage1.mybluemix.net/#/properties#--code) -
![Screen Shot 2021-02-22 at 12 38 32 PM](https://user-images.githubusercontent.com/25124000/108767157-e9595280-750a-11eb-93ce-8889944ff84b.png)

**Note -** 
There's an a11y violation in CodeMirror -
`Form control element <textarea> has no associated label`
This violation will be fixed by CodeMirror team in [issue 217](https://github.com/scniro/react-codemirror2/issues).
